sources/lib/components/station_ui/html/card.ex

```diff
@@ -171,8 +147,15 @@ defmodule StationUI.HTML.Card do
 
     assigns = assign(assigns, :class, class)
 
+    content_class =
+      if assigns.direction == "horizontal" do
+        [base_horizontal_content_classes() | List.wrap(class)]
+      else
+        [base_content_classes() | List.wrap(class)]
+      end
```

Comment:
Minor suggestion for consistency with the previous case statement vs if statement.
```suggestion
      case assigns do
        %{direction: "horizontal"} -> [base_horizontal_content_classes() | List.wrap(class)]
        _ -> [base_content_classes() | List.wrap(class)]
      end
```